docs/pages/kubernetes-access/guides/standalone-teleport.mdx

```diff
@@ -3,9 +3,11 @@ title: Kubernetes Access from a Standalone Teleport Cluster
 description: Connecting standalone Teleport installations to Kubernetes clusters.
 ---
 
-Teleport can connect to external Kubernetes clusters without using the Teleport
-Helm chart or running a pod inside of the Kubernetes cluster. To do this,
-Teleport needs a `kubeconfig` file to authenticate against the Kubernetes API.
+Teleport can connect to a Kubernetes cluster without using the Teleport Helm
+chart or running a pod inside of the Kubernetes cluster. To do this, you can run
+the Teleport Kubernetes Service on a Linux host. The Teleport Kubernetes Service
```

Comment:
Is there any way we could word "Teleport Kubernetes Service on a Linux host" to make it clearer that this host is not part of the cluster ?